nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java

```diff
@@ -71,7 +73,7 @@ private CommentUtils() {
      * @param expectedCommentedDocModel the document being commented, cannot be {@code null}
      */
     public static void checkDocumentEventContext(Event event, DocumentModel expectedCommentDocModel,
-            DocumentModel expectedCommentedDocModel) {
+            DocumentModel expectedCommentedDocModel, DocumentModel expectedTopLevelDocumentModel) {
```

Comment:
Can you add some javadoc please? It could worth it as we're having long discussion on what things are.